templates/tftmpl/variables.go

```diff
@@ -7,11 +7,16 @@ import (
 	"strings"
 
 	"github.com/hashicorp/consul-terraform-sync/templates/hcltmpl"
+	goVersion "github.com/hashicorp/go-version"
 	"github.com/hashicorp/hcl/v2/hclsyntax"
 	"github.com/hashicorp/hcl/v2/hclwrite"
 	"github.com/zclconf/go-cty/cty"
 )
 
+// tfVersionSensitive is the first version to support the sensitive argument
+// for variables
+var tfVersionSensitive = goVersion.Must(goVersion.NewSemver("0.14.0"))
```

Comment:
Will this version need to be adjusted at some point or will it become a legacy thing (ie. when we're on version 0.42.0 a few years down the road and no one uses anything this old we'd just remove it)? If we want to have a minimal version we support then maybe store these values in an obvious, common place... maybe a requirements.go or something?

Reply:
This version is only relevant to sensitive vars and not the Terraform version constraint supported. The supported Terraform version exists in version/terraform.go

I would much prefer to keep this variable where it's used since it only makes sense in the context of this file.

Reply:
I'd like to follow the pattern to support version n-2. And so if 0.16 is a thing or if Terraform 1.0s, then this can be dropped and assumed always sensitive